package.json

```diff
@@ -25,7 +25,7 @@
   "repository": "github:NikulinIlya/vue-easymde",
   "license": "MIT",
   "dependencies": {
-    "easymde": "^2.8.0",
+    "easymde": "^2.10.0",
     "marked": "^0.7.0"
```

Comment:
Easymde itself requires `marked`, so I'm not sure if this is needed here. Is there a special reason to include this?